sql-ballerina/tests/complex-query-test.bal

```diff
@@ -273,6 +275,105 @@ function testDateTime() {
     test:assertEquals(value, expected, "Expected record did not match.");
 }
 
+type ResultDates2 record {
+    time:Date DATE_TYPE;
+    time:TimeOfDay TIME_TYPE;
+    time:Civil TIMESTAMP_TYPE;
+    time:Civil DATETIME_TYPE;
+    time:TimeOfDay TIME_TZ_TYPE;
+    time:Civil TIMESTAMP_TZ_TYPE;
+};
+
+@test:Config {
+    groups: ["query", "query-complex-params"]
+}
+function testDateTime2() {
+    MockClient dbClient = checkpanic new (url = complexQueryDb, user = user, password = password);
```

Comment:
Shall we remove `checkpanic` and returns `error?` in function signature

Reply:
I will update